config/rubocop.yml

```diff
@@ -19,15 +19,14 @@ Style/StringLiterals:
 Metrics/LineLength:
   Max: 100
```

Comment:
I would be fine with you changing this to 80 as long as you add a note above the excludes that says don't bother updating those offending lines just for the sake of resolving line length violations. I think it is a good quality filter for new code and code that needs to be otherwise refactored but I don't want it to be viewed as a "TODO" like some other rubocop exclusions.